### ISSUE TITLE
Issue Summary

### ISSUE THREAD

**Issue by LelouBil** [JannisX11/blockbench#2567]
### What are you trying to do, and what do you expect to happen?

While editing a file in the "Java Block/Item" format, I disable a group of elements from the export.
I then save the file.
I expect a warning that my "disabled" elements are in fact completely absent in the "save" because for this format, saving is exporting

### What happens instead?

I get no warning, and can close and reopen blockbench at a later time and find my elements missing

### Model format in which the issue occurs

Java Block/Item

### Blockbench variant

Program

### Blockbench version

4.11.2

### Operating System

Windows 10

**Comment by LelouBil** [comment]
I guess this also applies to any properties that the format doesn't support. Hiding/showing elements also doesn't persist on save, due to the format. Blockbench should maybe disable these features for formats that don't support them (or show a warning for features that do not actually change data, like visibility, that it will not be persisted when saving) but I think disabling from export should not be available in file formats that don't support it

**Comment by JannisX11** [comment]
You have to keep in mind that people can still save project files which include all of this data, including visibility.

**Comment by LelouBil** [comment]
Of course, but Blockbench knows the current format of the file so I think a warning is at least needed.
